test/parentchild.js

```diff
@@ -1,362 +0,0 @@
-'use strict'
```

Comment:
Removing this test doesn't quite seem right.  Parent-child relationships are still a thing... they just can't be in the same index any more?

Reply:
ES 6 dropped support for parent-child (notice the tests failing). [parent-join](https://www.elastic.co/guide/en/elasticsearch/reference/6.0/parent-join.html) has to be used now.